src/test/java/seedu/duke/component/CapacitorTest.java

```diff
@@ -0,0 +1,30 @@
+package seedu.duke.component;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.DukeException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CapacitorTest {
+
+    @Test
+    void getValueTest() {
```

Comment:
Consider updating the method name to be more specific? Like https://se-education.org/guides/conventions/java/basic.html#naming
> `featureUnderTest_testScenario_expectedBehavior()`

Also can consider adding edge cases? Like 0 or negative values to make the tests more robust?